x-pack/plugins/security_solution/cypress/integration/network/hover_actions.spec.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  ADD_TO_TIMELINE,
+  CLOSE_TOP_N,
+  COPY,
+  EXPAND_OVERFLOW_ITEMS,
+  FILTER_IN,
+  FILTER_OUT,
+  SHOW_TOP_FIELD,
+  TOP_N_CONTAINER,
+} from '../../screens/network/flows';
+import { TOASTER } from '../../screens/configure_cases';
+import { GLOBAL_SEARCH_BAR_FILTER_ITEM } from '../../screens/search_bar';
+import { DATA_PROVIDERS } from '../../screens/timeline';
+import { cleanKibana } from '../../tasks/common';
+
+import { loginAndWaitForPage } from '../../tasks/login';
+import { NETWORK_URL } from '../../urls/navigation';
+import { TIMELINE_TOGGLE_BUTTON } from '../../screens/security_main';
+
+const testDomain = 'endpoint-dev-es.app.elstc.co';
+
+describe('Hover actions', () => {
+  beforeEach(() => {
+    cleanKibana();
```

Comment:
`cleanKibana` removes and adds again all the kibana indexes, so this method slows down the execution of the tests. Taking into account that these tests does not modify any kibana index, there is no need of executing it between tests. Just doing it one time is enough. So it is better to extract it to a `before` hook.

Doing it like this in my local the execution time of this spec goes from 123.27 to 80.65